omod/pom.xml

```diff
@@ -67,7 +67,7 @@
         <dependency>
             <groupId>org.openmrs.module</groupId>
             <artifactId>fhir2-omod</artifactId>
-            <version>1.4.0-SNAPSHOT</version>
+            <version>1.5.0-SNAPSHOT</version>
```

Comment:
@SanoferSameera 1.5.0 is released. You can change from SNAPSHOT to release version

Reply:
done